internal/graph/reachableresources.go

```diff
@@ -84,59 +98,58 @@ func (crr *ConcurrentReachableResources) ReachableResources(
 		return err
 	}
 
-	t := NewTaskRunner(ctx, crr.concurrencyLimit)
-
 	// For each entrypoint, load the necessary data and re-dispatch if a subproblem was found.
-	for _, entrypoint := range entrypoints {
-		switch entrypoint.EntrypointKind() {
-		case core.ReachabilityEntrypoint_RELATION_ENTRYPOINT:
-			err := crr.lookupRelationEntrypoint(ctx, t, entrypoint, rg, reader, req, stream, dispatched)
-			if err != nil {
-				return err
-			}
+	return withIterableInCursor(ci, "entrypoint", entrypoints,
```

Comment:
Entrypoints passed to this function come with an order, which is inferred by the reachability graph code by analyzing the schema. For example a `permission` has a specific order of the various entrypoints.

What happens if a call to `WriteSchema` happens that changes the orders of, say, a union based permission? E.g.

```
permission a = b + c +d
```

becomes 

```
permission a = c + d + b
```

and someone happens to be in the middle of paginating a very large response, I'm the middle of which the schema (and reachability graph) changed. Would we get a consistent view of this because the cursor has selected a consistent snapshot of the state of the system, including the schema?

Reply:
Yes, because it encodes the revision. Same as with a check being dispatched during a schema change